control-plane/grpc/proto/v1/ha/cluster_agent.proto

```diff
@@ -26,9 +26,10 @@ message FailedNvmePath {
 message ReportFailedNvmePathsRequest {
   // Node which reports failed paths.
   string nodename = 1;
-
-  // List of failed
-  repeated FailedNvmePath failed_paths = 4;
+  // List of failed paths.
+  repeated FailedNvmePath failed_paths = 2;
+  // Endpoint of the node agent.
+  string endpoint = 3;
```

Comment:
nit: keep endpoint next to nodename since they're more closely related?

Reply:
Incorporated